Source/WebCore/platform/animation/AcceleratedEffectStack.h

```diff
@@ -60,12 +60,10 @@ class AcceleratedEffectStackBase {
     
     WEBCORE_EXPORT void initAsyncEffects(PlatformLayer *, Seconds);
     WEBCORE_EXPORT void applyBackdropLayerEffects(PlatformLayer *, Seconds) const;
-    WEBCORE_EXPORT void applyEffectsAsync(const FloatRect&, Seconds) const;
     WEBCORE_EXPORT void clear(PlatformLayer *);
```

Comment:
All of those should move to the `Remote` class. We probably want to remove the `Async` naming there. Fine to do in another patch of course.